ParisBNF/abb/BNFabb30.xml

```diff
@@ -0,0 +1,720 @@
+                  space was solved by means of <locus from= "202r" to="202v"> where
+                    the scribe increased the size of his handwriting and the distance
+                  between the letters (ca. 11 to 13 per line).</locus></note>
+                             </handNote>
+
+                      <handNote script="Ethiopic" xml:id="h2" corresp="#a4">
+                          <date notBefore="1772" notAfter="1773"/>
+                        </handNote>
+                      </handDesc>
+
+
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1r"/>
+                               <desc type="RecordTransaction"/>
+                                 <q xml:lang="gez">ኍልቈ᎓ እህል᎓ ዘወጽአ᎓ በዘመነ᎓ ሉቃስ᎓></q>
+                                 <note>The note concerns tributes or transaction with cereals.</note>
+                                 </item>
+
+                                  <item xml:id="a2">
+                                    <locus from="1r" to="2r"/>
+                                     <desc type="GuestText" corresp="LIT2872RepCh153">A
+                                       malkǝʾ-hymn for the Four Living Creatures.</desc>
+                                      <q xml:lang="gez"></q>
+                                     </item>
+
+                                     <item xml:id="a3">
+                                       <locus target="#3r"/>
+                                        <desc type="OwnershipNote"/>
+                                          <q xml:lang="gez">ይትባርክ᎓ እግዚአብሔር፡ አምላከ᎓ እስራኤል᎓ ዝመጽሐፍ᎓
+                                            ለአባ᎓ አሕዋን᎓ no. 30</q>
+                                           <note>The note is written in a crude hand, in the upper margin,
+                                             in two column. It mentions an owner,
+                                             <persName ref="PRS12914Ahwan" role="owner">the Falāšā manʾabbā ʾAḥwān</persName> from whom
+                                             <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
+                                             bought the manuscript. The number 30 could have been added later.</note>
+                                         </item>
+
+                                         <item xml:id="a4">
+                                           <locus from="#202va" to="203ra"/>
+                                            <desc type="GuestText"/>
+                                              <q xml:lang="gez"></q>
+                                              <note>The note appears as a continuation of <ref type="mspart" corresp="ms_i8"/>,
+                                                added at a later time, in a less professional hand (the <foreign>tamallas</foreign>-sign
+                                                sing inticates that it goes on on the next page).
```

Comment:
sign indicates, maybe "continues" to prevent the two on?

Reply:
Changed

Reply:
Changed.